sheet/ozymandias/Aldin.qml

```diff
@@ -0,0 +1,501 @@
+                Bonus {
+                    source: toughness
+                    amount: Math.max(3, level.value)
+                },
+                Bonus{
+                    name: "Level 2 Health Roll"
+                    amount: 5
+                },
+                Bonus{
+                    name: "Level 3 Health Roll"
+                    amount: 7
+                }
+
+            ]
+        }
+
+        Attribute {
+            id: currentHp
+            name: "Current Hit Points"
+            uri: hp.prefix + "/current"
+
+            modifiers: [
+                Bonus {
+                    name: "Initial Value (Max)"
+                    amount: maxHp.value
+                }
+            ]
+            readOnly: false
+        }
+    }
+
+    Attribute {
+        id: armourClass
+        name: "Armour Class"
+        uri: "attr://combat/armourClass"
+
+        modifiers: [
+            Bonus {
+                name: "Base"
+                amount: 10
+            },
+            Bonus {
+                id: acDexBonus
+                name: "Dexterity"
+                amount: dexterity.temporary.modifier.value
+            },
+            Bonus {
+                id: acArmourBonus
+                name: "Armour (Mage Armor)"
+                amount: 4
+            },
+            Bonus {
+                id: acShieldBonus
+                name: "Buckler"
+                amount: 1
+            },
+            Bonus {
+                id: acDeflectionBonus
+                name: "Hawk Badge"
+                amount: 1
+            },
+
+            Bonus {
+                name: size.name
+                amount: -sizeBonus(size.value)/2
+            }
+
+        ]
+
+        Attribute {
+            name: "Armour Class (Touch)"
+            uri: parent.uri + "/touch"
+            modifiers: [
+                Bonus {
+                    name: "Normal Armour Class"
+                    amount: armourClass.value
+                },
+                Bonus {
+                    name: "no armor"
+                    amount: -acArmourBonus.amount
+                },
+                Bonus {
+                    name: "no shield"
+                    amount: -acShieldBonus.amount
+                }
+            ]
+        }
+
+        Attribute {
+            name: "Armour Class (Flat-Footed)"
+            uri: parent.uri + "/flatFooted"
+
+            modifiers: [
+                Bonus {
+                    name: "Normal Armour Class"
+                    amount: armourClass.value
+                },
+                Bonus {
+                    name: "no dex"
+                    amount: -acDexBonus.amount
+                }
+            ]
+        }
+    }
+
+    Attribute {
+        name: "Initiative"
+        modifiers: [
+            Bonus {
+                name: dexterity.temporary.modifier.name
+                amount: dexterity.temporary.modifier.value
+            }
+        ]
+    }
+    Attribute {
```

Comment:
Is this actually needed anymore, since Ethan ruled that the bonus you get is from the spell/effect that changes your size, rather than the inherent act of changing size?